src/bindings/crypto/napi-conversion-core.ts

```diff
@@ -31,9 +37,42 @@ function napiConversionCore(napi: any) {
     PolyComm: napi.WasmFqPolyComm,
   });
 
+  const shared = {
```

Comment:
Mmmm this was not compiling for me if this was part of the `shared`, that's why I moved it out...